kafka-plugins-0.10/src/main/java/io/cdap/plugin/source/KafkaConfig.java

```diff
@@ -81,14 +84,19 @@
   @Macro
   private String initialPartitionOffsets;
 
-  @Description("The default initial offset for all topic partitions. " +
-    "An offset of -2 means the smallest offset. An offset of -1 means the latest offset. Defaults to -1. " +
-    "Offsets are inclusive. If an offset of 5 is used, the message at offset 5 will be read. " +
-    "If you wish to set different initial offsets for different partitions, use the initialPartitionOffsets property.")
+  @Description("The default initial offset for all topic partitions. Defaults to -1.")
```

Comment:
offset = -1, -2 is meaningless to the user. Can you please check to make sure that references to -1, -2 don't show up in user visible messages/documentation?

Reply:
Would be great if you can add screenshots of the widget configuration and documentation page. Thanks!

Reply:
> offset = -1, -2 is meaningless to the user. Can you please check to make sure that references to -1, -2 don't show up in user visible messages/documentation?

Added missing documentation for the new property ```initialOffset``` and updated it's description. However did not change documentation of ```defaultInitialOffset``` because this part still valid in scenario where user chooses ```Start from specific offset``` option for initial offset then the widget for default initial offset is shown where the existing documentation is needed.

Reply:
> Would be great if you can add screenshots of the widget configuration and documentation page. Thanks!

![339-documentation](https://user-images.githubusercontent.com/7079754/112292452-622a0680-8c91-11eb-9331-64c94b3b4ff5.png)
![339-last-proccessed](https://user-images.githubusercontent.com/7079754/112292466-66eeba80-8c91-11eb-9b8e-cc351db414da.png)
![339-specific-offset](https://user-images.githubusercontent.com/7079754/112292480-6a824180-8c91-11eb-8c6e-34fb067e67c8.png)